Added validate of annualSalary

BregornOriginal opened the pull request.

BregornOriginal committed:
Added validate of annualSalary

fsodano commented on clase-6/tarea-2-calculos.js:
```diff
@@ -0,0 +1,44 @@
+  };
+  
+  function calculateAverageAnnualSalary(arraySalary) {
+    let arraySalaryNoZero = [];
```
`validSalaries`

BregornOriginal replied:
Ok!

fsodano commented on clase-6/tarea-2-calculos.js:
```diff
@@ -0,0 +1,44 @@
+function calculateMaxSalary(arraySalary) {
```
`salaries`, si es plural ya sabés que es un array, no te hace falta explicitar que es un array.

BregornOriginal replied:
Dale Fabri, lo corrijo!

fsodano commented on clase-6/tarea-2-calculos.js:
```diff
@@ -0,0 +1,44 @@
+    return averageAnnualSalary;
+  };
+  
+  function calculateAverageMensualSalary(averageAnnualSalary) {
```
monthly

BregornOriginal replied:
Me descuidé al traducir!

fsodano commented on clase-6/tarea-2-pruebas.js:
```diff
@@ -0,0 +1,12 @@
+    	"Validate Annual Salary didn't validate that salary is greater than 0"
+    );
+    console.assert(
+        validateAnnualSalary("letras") === "This field only accept numbers",
```
ojo con el spanglish

fsodano commented on clase-6/tarea-2.js:
```diff
@@ -28,14 +26,20 @@ document.querySelector("#add-member").onclick = function (event) {
-  $newInput.setAttribute("id", "members-annual-salary");
-  $newInput.setAttribute("placeholder", "Annual salary");
-  $newInput.value = 0;
+  inputNumber = document.querySelectorAll(
```
dónde está la declaración de inputNumber?

BregornOriginal replied:
Ya le agregó el tipo de variable

fsodano commented on clase-6/tarea-2.js:
```diff
@@ -45,58 +49,58 @@ const $buttonRemoveMember = document.getElementById("remove-member");
+function getAnnualSalaryOfMember() {
+  
+  let arraySalary = [];
+  let inputAnnualSalary = inputNumber = document.querySelectorAll(
```
esto no me cierra che, qué quisiste hacer acá?

BregornOriginal replied:
La verdad..... No sé jajaja.... lo voy a dejar así 

let inputNumber = document.querySelectorAll(
    "#container-annual-salary input"
  );
  console.log(document.querySelectorAll("#container-annual-salary input"));
  for (let i = 0; i < inputNumber.length; i++) {
    if (inputNumber[i].value != "") {
      arraySalary.push(Number(inputNumber[i].value));
    }
  }

BregornOriginal committed:
Added description errors

Now when you get an error on validatiation input, a message show you the error you have done.
Now a border red, at input error, show correctly if an error exist or not.

Added a new validation for input salaries, as "", this one have to by checked.

Minnor improvementes:
- Changed name of variables for better names.
- Translated words were corrected.
- Console log deleted.

BregornOriginal commented:
Merge
